docs/modules/ROOT/pages/http-client.adoc

```diff
@@ -680,6 +680,50 @@ include::{examples-dir}/channeloptions/Application.java[lines=18..54]
 <4> Configures the time between individual `keepalive` probes to 1 minute.
 <5> Configures the maximum number of TCP `keepalive` probes to 8.
 
+
+==== IdleTimeout
+The maximum time (resolution: ms) that this connection stays opened and waits for HTTP request. Once the timeout is reached, the connection is closed.
```

Comment:
> The maximum time (resolution: ms) that this connection stays opened and waits for HTTP request

Can you please elaborate more on this?

This is HttpClient so it initiates the HTTP request. Also as part of the connection pool we already have a configuration for dropping the idle connections so this new configuration seems to be not needed?

Reply:
@violetagg 
This describes configuring `IdleTimeout` on the HttpClient side using `IdleTimeoutHandler`, independently of the connection pool settings.

From my understanding, since the connection pool already manages idle connections, applying an additional `IdleTimeoutHandler` at the `HttpClient` level may be redundant. 
Could you please confirm if this understanding is accurate?

Reply:
yes I think it is redundant